webpack.config.js

```diff
@@ -0,0 +1,27 @@
+const path = require('path');
+
+module.exports = {
+  mode: "none",
+  watch: true,
+  entry: './src/js/index.js',
+  output:{
+     path: path.resolve(__dirname, "dist"),
+     filename: "bundle.js",
+     publicPath: path.resolve(__dirname, 'src')
+  },
+  module:{
+   rules:[
+     {
+         test: /\.(jpe?g|png|gif|svg)$/i,
+         use: [{
+             loader: 'url-loader',
+             options: {
+               limit: 8000,
+               name: '[name].[ext]',
+               publicPath: 'src/assets/img/'
```

Comment:
Let us try to simplify this path. As it is right now when you open the generated `dist` folder we have a `dist/img` and then another `dist/src/img` folder. The contents of both are actually exactly the same.